controls/os_spec.rb

```diff
@@ -236,3 +238,23 @@
     its(:group) { should match(/^root|syslog$/) }
   end
 end
+
+control 'os-12' do
+  impact 1.0
+  title 'Detect cpu-vulnerability-directory'
```

Comment:
Can you please rename the title to something like "Detect vulnerabilities in the cpu-vulnerability-directory"?

Reply:
Changed Thanks